test/benchmark.py

```diff
@@ -62,15 +62,32 @@ def human_int(value):
     return int(value) * multiplier
 
 
-def create_test_file(path, max_size, verbose=False):
+def create_test_file(path, max_size, verbose=False, chunk_size=65535):
+    if max_size < 1:
+        raise ValueError("max_size must be >= 1")
+    if chunk_size < 1:
+        raise ValueError("chunk_size must be >= 1")
     size = random.randint(1, max_size)
     if verbose:
         print('creating', path, 'size', size)
-    with open(path, 'wb') as f:
-        # TODO: make this more efficient.
-        while size:
-            f.write(b'%c' % random.randint(0, 255))
-            size -= 1
+    
```

Comment:
There is still a flake8 error on this line.

```
Run flake8 --exclude=build,venv --ignore= --max-line-length=200 --max-complexity=75 --show-source --statistics .
./test/benchmark.py:73:1: W293 blank line contains whitespace
    
^
1     W293 blank line contains whitespace
Error: Process completed with exit code 1.
```

See: https://github.com/cfv-project/cfv/actions/runs/18927372363/job/54136536126?pr=84

```suggestion

```